src/gamekit.cs

```diff
@@ -2456,4 +2629,91 @@ interface GKSessionDelegate {
 		[Export ("session:didFailWithError:")]
 		void FailedWithError (GKSession session, NSError error);
 	}
+
+	[TV (14,0), Mac (10,16), iOS (14,0)]
+	[NoWatch]
+	[BaseType (typeof (NSObject))]
```

Comment:
`[DisableDefaultCtor]` since there's a `shared` method
unless docs/headers mention you can create your own ?!?

Reply:
Doesn't look like you can create one 👍